torchfields/voting.py

```diff
@@ -80,13 +87,45 @@ def vote(self, softmin_temp=1, blur_sigma=1):
     mavg = torch.stack(mtuple_avg)
 
     # compute weights for mtuples: smaller mean distance -> higher weight
-    mt_weights = (-mavg/softmin_temp).softmax(dim=0)
+    # computing softmin explicitly for access to numerator & denominator
+    # equivalent to:
+    # (-mavg/softmin_temp).softmax(dim=0) == mt_weights_num / mt_weights_den
+    mt_weights_num = torch.exp(-mavg/softmin_temp)
+    mt_weights_den = mt_weights_num.sum(dim=0, keepdim=True)
```

Comment:
It says you're computing the softmin explicitly for access to the numerator and denominator, but the numerator and denominator aren't being returned to the caller, right? They're only being used to immediately calculate `mt_weights`, so it doesn't seem like they are made available.
Am I missing something?